contributing_to_docs/term_glossary.adoc

```diff
@@ -103,11 +103,21 @@ action. It consists of _identity_ and _action_.
 ''''
 === build
 
-General usage: build(s)
+Usage: build(s), or when speaking generally about `Build` objects.
 
-A build is the process of transforming input parameters into a resulting object.
+* A build is the process of transforming input parameters into a resulting object.
+* A `Build` object encapsulates the inputs needed to produce a new deployable image, as well as the status of the execution and a reference to the pod which executed the build.
 
-When referencing the actual object, write as `Build`. See link:doc_guidelines.adoc#api-object-formatting[API object formatting] for more details.
+When referencing the actual object, write as `Build` as appropriate. See link:doc_guidelines.adoc#api-object-formatting[API object formatting] for more details.
```

Comment:
It would be helpful to be more specific (both here and in the same line for "build configuration") in the usage example. Something like this:

> When referencing the actual object, write "`Build` object" as appropriate.

The way you have it now is correct, but hopefully the additional guidance would prevent people from forgetting the "object" part in such cases.

Reply:
Fun fact that I had to look up: to maintain the monospace for `Build` and `BuildConfig` within quotations, you have to add a second set of back ticks. For example, "``Build`` object."